src/main/java/org/mozilla/msrp/platform/vertical/video/YoutubeClientConfig.kt

```diff
@@ -0,0 +1,16 @@
+package org.mozilla.msrp.platform.vertical.video
+
+import com.google.api.client.http.javanet.NetHttpTransport
+import com.google.api.client.json.jackson2.JacksonFactory
+import com.google.api.services.youtube.YouTube
+import org.springframework.context.annotation.Bean
+import javax.inject.Named
+
+@Named
+class YoutubeClientConfig {
+    @Bean
+    fun youtubeClientFactory(): YouTube {
+        return YouTube(NetHttpTransport(), JacksonFactory()) {
```

Comment:
I'm not sure what this is

Reply:
Instantiate a `Youtube` API instance to be used in the repository